tests/ui/suggestions/issue-117669.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    let abs: i32 = 3i32.checked_abs();
```

Comment:
You need to add an error expectation here:

```suggestion
    let abs: i32 = 3i32.checked_abs();
    //~^ ERROR mismatched types
```

Reply:
Caught very quickly, thanks!